src/main/java/core/basesyntax/service/strategy/FruitStrategy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dto.FruitTransactionDto;
+import java.util.List;
+
+public class FruitStrategy {
+    private final List<OperationHandler> handlers;
+
+    public FruitStrategy(List<OperationHandler> handlers) {
+        this.handlers = handlers;
+    }
+
+    public List<OperationHandler> getHandlers(FruitTransactionDto dto) {
+        return handlers.stream()
+                .filter(h -> h.isApplicable(dto))
+                .toList();
+    }
+}
```

Comment:
I remember during the lesson with the coach, we were exploring applicable handlers like this, but let's simplify. In this specific task, we have only one handler for each operation type, and every time we receive only one handler. Therefore, there's no reason to put it into a list.

Reply:
Agree, not resolved, we do not need this `isApplicable()` method. 
Use`Map<Operation, OperationHandler>` instead of this list to easily get handler by operation